examples/panel-datalinks/src/plugin.json

```diff
@@ -8,7 +8,10 @@
     "author": {
       "name": "My org"
     },
-    "keywords": ["panel", "datalinks"],
+    "keywords": [
```

Comment:
@mckn there's something in your editor configuration not picking up the prettier configuration.  I notice in this (and other files) there are some changes in quotes and spaces.

In my case if I run prettier in this file I get `"keywords": ["panel", "datalinks"],` (all in one line) vs here in separated lines. 

Could you verify what is the result if you run prettier in CLI against this file?

Reply:
I too get the same result as @academo when saving the plugin.json. I've gone ahead and updated example files with `prettier -w`.

Reply:
Somehow I got confused and thought @mckn was the PR author. Thanks for checking!